src/SQLStore/EntityStore/TraversalPropertyLookup.php

```diff
@@ -73,46 +71,39 @@ public function fetchFromTable( PropertyTableDef $propertyTableDef, DataItem $da
 			// from the inner join
 			$options['GROUP BY'] = 'p_id';
 
-			$opt = OptionsBuilder::toString( $options );
+			$subquery = $builder->newSubquery()
+				->select( 'p_id' )
```

Comment:
@coderabbitai this is throwing "Error: Call to a member function select() on null", any ideas why and how to fix?

Reply:
That doesn't fix it and it happens with MediaWiki 1.39 and that supports SelectQueryBuilder with select()